lock-api/src/main/java/com/palantir/lock/LockServiceAdapter.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.lock;
+
+import java.math.BigInteger;
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+public class LockServiceAdapter implements LockService {
```

Comment:
so since `LockServiceAdapter` implements `LockService`, it's already implemented `RemoteLockServiceAdapter`.

People can't cast at the moment, since `LockService` and `RemoteLockService` are proxies, that said they'd have to notice this change in order to have changed their behaviour, so I think it's safe to get rid of the `RemoteLockServiceAdapter`.

Reply:
How do you get a RemoteLockService from a RemoteRpcClient?